docs/data-sources/scim_group.md

```diff
@@ -0,0 +1,25 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "temporalcloud_scim_group Data Source - terraform-provider-temporalcloud"
+subcategory: ""
+description: |-
+  Fetches details about a SCIM group.
+---
+
+# temporalcloud_scim_group (Data Source)
+
+Fetches details about a SCIM group.
+
+
```

Comment:
Can you add an example here: https://github.com/temporalio/terraform-provider-temporalcloud/tree/main/examples/data-sources

That'll make the docs much more helpful